main.go

```diff
@@ -647,6 +648,26 @@ func loadConfig(filename string, queryAWSUserData bool, queryGCPMetaData bool) (
 	return c, nil
 }
 
+var exposer expose.Exposer
+
+func setupExposer() (err error) {
+	if config.LiveLogSecret != "" {
```

Comment:
Is it possible to put this in the livelog feature?

Reply:
No, as I mentioned i the commit message, this isn't a livelog secret so much as a stateless-dns secret.  Since stateless-dns will go away soon, I didn't see the use in a breaking change to rename the g-w config field from `liveLogSecret` to `statelessDNSSecret`.

Reply:
ok!